examples/parallel-reconstruct.py

```diff
@@ -0,0 +1,170 @@
+import shutil
+import os
+import numpy as np
+import multiprocessing as mp
+from wget import download
+from datetime import datetime
+from iohub import open_ome_zarr
+from iohub.convert import TIFFConverter
+from recOrder.io.utils import load_bg
+from recOrder.compute.reconstructions import (
+    initialize_reconstructor,
+    reconstruct_qlipp_stokes,
+    reconstruct_qlipp_birefringence,
+    reconstruct_phase3D,
+)
+
+# This example demonstrates a 3D-pol-to-birfringence reconstruction applied
+# to each position and time point of a micromanager dataset. The example
+# uses multiprocessing to apply the reconstructions in parallel.
+
+# This example will download a ~50 MB test dataset to your working directory.
+
+N_processes = 6  # (should have at least this many cores available)
+
+
+def precomputation():
+
+    # download test data
+    data_folder = os.path.join(os.getcwd(), "data_temp")
+    os.makedirs(data_folder, exist_ok=True)
+    url = "https://zenodo.org/record/6983916/files/recOrder_test_data.zip?download=1"
+    zip_file = "recOrder_test_Data.zip"
+    output = os.path.join(data_folder, zip_file)
+    if not os.path.exists(output):
+        print("Downloading test files...")
+        download(url, out=output)
+        shutil.unpack_archive(output, extract_dir=data_folder)
+
+    temp_path = os.path.join(data_folder, "temp.zarr")
+    converter = TIFFConverter(
+        os.path.join(
+            data_folder,
+            "2022_08_04_recOrder_pytest_20x_04NA/2T_3P_16Z_128Y_256X_Kazansky_1",
+        ),
+        temp_path,
+    )
+    converter.run()
+
+    # setup input reader
+    reader = open_ome_zarr(temp_path, mode="r+")
+    T, C, Z, Y, X = reader["0/0/0"].data.shape
+    dtype = reader["0/0/0"].data.dtype
+    P = len(list(reader.positions()))
+    bg_data = load_bg(
+        os.path.join(data_folder, "2022_08_04_recOrder_pytest_20x_04NA/BG/"),
+        height=Y,
+        width=X,
+    )
+
+    # setup output zarr
+    timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+    writer = open_ome_zarr(
```

Comment:
This script cannot be ran repeatedly without manual editing or removing previous data since the conversion cannot (and maybe should not) overwrite. So is it necessary to write to two different stores for conversion and reconstruction?

Reply:
Good point. I just made a change to convert to a zarr store with a timestamped name instead of overwriting so that we can rerun the script. 

I know this isn't as space efficient as overwriting the converted store, but I think the clarity is worth it. 

I also added a comment mentioning that you should only use this download-and-convert section for a test run.